packages/design-picker/src/components/style-variation-badges/utils.ts

```diff
@@ -1,31 +1,11 @@
 import type {
 	StyleVariation,
 	StyleVariationSettingsColorPalette,
-	StyleVariationStylesColor,
 	StyleVariationPreview,
 	StyleVariationPreviewColorPalette,
 } from '../../types';
 
-const COLOR_PALETTE_SUPPORTS = [ 'background', 'foreground', 'primary', 'secondary', 'tertiary' ];
-
-const GLOBAL_STYLES_SUPPORTS: {
-	style: keyof StyleVariationStylesColor;
-	infix: 'color';
-	mapTo: keyof StyleVariationPreviewColorPalette;
-}[] = [
-	{ style: 'background', infix: 'color', mapTo: 'background' },
-	{ style: 'text', infix: 'color', mapTo: 'foreground' },
-];
-
-function parsePresetDeclaration( preset: string, infix: string ): string | undefined {
-	return preset.match( `--wp--preset--${ infix }--([a-z]+)` )?.[ 1 ];
-}
-
-function getValueFromVariationStylesColor(
-	variation: StyleVariation
-): StyleVariationStylesColor | undefined {
-	return variation.styles?.color;
```

Comment:
I think we can also remove this field from the type:

https://github.com/Automattic/wp-calypso/blob/4033371065bfc50a48909acca50d64e95340a1e3/packages/design-picker/src/types.ts#L28

Reply:
@fushar I forgot that we need this field, for when we fetch the full `theme.json` from the `v2/starter-designs/${theme_slug}` API, which will be fed to the style switcher buttons.

Reply:
Ah, right... OK OK 😅